### PR TITLE
Pal 127 create project structure for repo

### PR DESCRIPTION
**Pull request by dev-958**

### CODE REVIEW

**Review by ac74475 — COMMENTED** [review]
Can we update the year in the copyright to 2019?
Can we update the starting version number for all the services to 0.4.0?
Do we need the Logos in this repo or is it a better fit to have them in the documentation repo?

**Review by nw1984 — DISMISSED** [review]

File: charts/audit-service/values.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+replicaCount: 1
```
Comment (ac74475) [inline charts/audit-service/values.yaml:15]: Can we add the Copyright and licence header to this file

File: audit-service/src/main/resources/values.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+replicaCount: 1
```
Comment (ac74475) [inline audit-service/src/main/resources/values.yaml:15]: Can we add the Copyright and licence header to this file

File: audit-service/src/main/resources/bootstrap.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+spring:
```
Comment (ac74475) [inline audit-service/src/main/resources/bootstrap.yaml:15]: Can we add the Copyright and licence header to this file

File: audit-service/src/main/resources/application.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+spring:
```
Comment (ac74475) [inline audit-service/src/main/resources/application.yaml:15]: Can we add the Copyright and licence header to this file

File: audit-service/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,10 @@
+FROM openjdk:11
```
Comment (ac74475) [inline audit-service/Dockerfile:15]: Can we add the Copyright and licence header to this file

File: .helmignore [hunk 6]
```diff
@@ -0,0 +1,22 @@
+# Patterns to ignore when building packages.
```
Comment (ac74475) [inline .helmignore:15]: Can we add the Copyright and licence header to this file

File: discovery-service/src/main/resources/application.yaml [hunk 7]
```diff
@@ -0,0 +1,25 @@
+spring:
```
Comment (ac74475) [inline discovery-service/src/main/resources/application.yaml:15]: Can we add the Copyright and licence header to this file

File: discovery-service/src/main/resources/bootstrap.yaml [hunk 8]
```diff
@@ -0,0 +1,4 @@
+spring:
```
Comment (ac74475) [inline discovery-service/src/main/resources/bootstrap.yaml:15]: Can we add the Copyright and licence header to this file

File: user-service/pom.xml [hunk 9]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```
Comment (ac74475) [inline user-service/pom.xml:17]: Can we add the Copyright and licence header to this file

**Review by ac74475 — CHANGES_REQUESTED** [review]

**Review by ac74475 — APPROVED** [review]

**Pull request merged**